docs/ota-client-guide/modules/ROOT/pages/device-cred-prov-steps.adoc

```diff
@@ -1,6 +1,6 @@
 = Set Up Device Provisioning
 
-In this section, you'll learn how provisioning in production is different from the provisioning method that you used to get started. You'll also understand the major steps involved in provisioning devices for production.
+Learn how provisioning in production is different from the provisioning method that you used to get started. You'll also understand the major steps involved in provisioning devices for production.
```

Comment:
Second sentence here is still in the imperative mood; it makes the paragraph read funny to me.

Reply:
Ok, reverted back to original